app/views/merchant_site/invoices/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1> Merchant: <%= @merchant.name %> Invoices </h1>
```

Comment:
Because we haven't create the invoices page yet, this will throw an error in one of our tests, but it's empty right now